src/components/shared/Popover/Popover.tsx

```diff
@@ -45,6 +45,8 @@ interface Props {
   renderContentValues?: SimpleMessageValues;
   /** Set the open state from outside */
   isOpen?: boolean;
+  /** State setter to sync external state with internal state */
+  setIsOpen?: SetStateFn<boolean>;
```

Comment:
The changes in this file fix an issue with the external state of the popover (passed in via TokenActivationPopover) not being updated to reflect the internal state of the popover component